app/experimenter/nimbus-ui/src/types/getAllExperiments.ts

```diff
@@ -56,5 +59,8 @@ export interface getAllExperiments_experiments {
 }
 
 export interface getAllExperiments {
+  /**
+   * List Nimbus Experiments.
+   */
   experiments: (getAllExperiments_experiments | null)[] | null;
```

Comment:
do we want to include doc string for Api's????

Reply:
These files are auto generated, I guess the new version prints these out.